cli/go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/mitchellh/mapstructure v1.4.2
 	github.com/spf13/cast v1.4.1 // indirect
 	github.com/spf13/cobra v1.2.1
+	github.com/spf13/viper v1.9.0
```

Comment:
Latest version of Viper at the time of this PR
![image](https://user-images.githubusercontent.com/34534103/140332177-68604638-07af-41e5-a88f-6f1e9ecbe008.png)
https://pkg.go.dev/github.com/spf13/viper?tab=versions